portia/execution_agents/one_shot_agent.py

```
@@ -174,7 +179,22 @@ def execute_sync(self) -> Output:
         if not self.tool:
             raise InvalidAgentError("No tool available")
 
-        context = self.get_system_context()
+        # Note: the one-shot agent does not support pulling outputs from agent memory
```

Should this be an error? Seems like it could lead to weird behaviour and silent failures

---

Good call - I've now made it an error